Makefile

```diff
@@ -100,6 +100,7 @@ test-js:
 
 ensure:
 	dep ensure
+	cp scripts/patch/reflector.go.txt vendor/k8s.io/client-go/tools/cache/reflector.go
```

Comment:
This seems pretty mysterious to a reader. How about comment explaining why it's there and/or a TODO saying to remove this once we get https://github.com/kubernetes/kubernetes/pull/78991? (I guess that'll be 1.16.0?)

Reply:
done